cluster/saltbase/salt/kube-scheduler/kube-scheduler.manifest

```diff
@@ -8,7 +8,10 @@
 {
 "apiVersion": "v1",
 "kind": "Pod",
-"metadata": {"name":"kube-scheduler"},
+"metadata": {
+  "name":"kube-scheduler",
+  "namespace": "kube-system"
```

Comment:
Not sure, no. But it seemed weird to have them in the default namespace, and kube-system is where we've been putting the system components. The master components are also system components, so that seemed like the logical place to move them to.